src/css/_genes.scss

```
@@ -0,0 +1,5 @@
+@mixin genes($theme-name, $theme-colors) {
+    .search-bar {
+      margin-bottom: .25rem
```

```suggestion
      margin-bottom: .25rem
      border: 0;
      padding: 2px;
      border-radius: 2px;
```

I think we should add a small border radius to match the border radius of the outer component